svcs/amigo/amigo.go

```diff
@@ -925,3 +940,46 @@ func parseTemplates(givenTemplate string) (*template.Template, error) {
 	)
 	return tmpl, err
 }
+
+func checkTeamName(input string) error {
+	re := regexp.MustCompile("^[A-Za-z0-9]+$")
+
+	if input == "" {
+		return ErrTeamNameEmpty
+	}
+
+	if err := checkVarLength(input, TeamNameMaxLength); err != nil {
+		return ErrTeamNameToLarge
+	}
+
+	if !re.MatchString(input) {
+		return ErrTeamNameCharacters
+	}
+
+	return nil
+}
+
+func checkEmail(input string) error {
+	re := regexp.MustCompile("^[a-zA-Z0-9.!#$%&'*+^`{|}~]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$")
+
+	if input == "" {
+		return ErrEmailEmpty
+	}
+
+	if err := checkVarLength(input, emailMaxLength); err != nil {
+		return ErrEmailToLarge
+	}
```

Comment:
I think it is forgotten, because it does not seem correct, `checkVarLength` is already returning an error however `ErrEmailToLarge` variable is passed as an error. 
`checckVarLength` should return `ErrEmailTooLarge` error at line 980.

Reply:
See comment bellow